scrapscript.py

```diff
@@ -702,10 +715,12 @@ def eval_list(env: Env, exp: Object) -> typing.List[Object]:
     BinopKind.MOD: lambda env, x, y: Int(eval_int(env, x) % eval_int(env, y)),
     BinopKind.EQUAL: lambda env, x, y: Bool(eval_exp(env, x) == eval_exp(env, y)),
     BinopKind.NOT_EQUAL: lambda env, x, y: Bool(eval_exp(env, x) != eval_exp(env, y)),
-    BinopKind.LESS: lambda env, x, y: Int(eval_int(env, x) < eval_int(env, y)),
-    BinopKind.GREATER: lambda env, x, y: Int(eval_int(env, x) > eval_int(env, y)),
-    BinopKind.LESS_EQUAL: lambda env, x, y: Int(eval_int(env, x) <= eval_int(env, y)),
-    BinopKind.GREATER_EQUAL: lambda env, x, y: Int(eval_int(env, x) >= eval_int(env, y)),
+    BinopKind.LESS: lambda env, x, y: Bool(eval_int(env, x) < eval_int(env, y)),
+    BinopKind.GREATER: lambda env, x, y: Bool(eval_int(env, x) > eval_int(env, y)),
+    BinopKind.LESS_EQUAL: lambda env, x, y: Bool(eval_int(env, x) <= eval_int(env, y)),
+    BinopKind.GREATER_EQUAL: lambda env, x, y: Bool(eval_int(env, x) >= eval_int(env, y)),
```

Comment:
if we are going to change all of these we should test those too :P maybe best to leave for another commit (add TODO)

dunno how much you are trying to take on rn

Reply:
Will add some more in follow up